src/UIKit/UIGestureRecognizer.cs

```diff
@@ -59,13 +58,14 @@ void OnDispose ()
 		//
 		public UIGestureRecognizer (Selector sel, Token token) : this (token, sel)
 		{
-			recognizers [token] = sel.Handle;
+
+			recognizers?.Add (token, sel.Handle);
```

Comment:
In particular this is a pretty big change in behavior, because `Add` will throw an exception if the key already exists in the dictionary (while the indexer will just overwrite the existing value if the key exists).